toolbox/util/gcsClient.go

```diff
@@ -44,12 +45,10 @@ func NewGCSClient() *GCSClient {
 func (gcs *GCSClient) GetFileFromGCSReader(bucket, obj string) (*storage.Reader, error) {
```

Comment:
rename to GetFile since its GCSClient it is implied.

Reply:
Done. In my defense Yutong commited this code lol